viz.yaml

```diff
@@ -265,15 +281,18 @@ publish:
       hashes: url_hashes
```

Comment:
would you mind renaming `social-section` to `social_section` and `national-water-use` to `national_water_use` while you're in here? the style police are really getting on my case about it =)

Reply:
on it!